app/controllers/registrations/user_profile_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
```

Comment:
Am I right in thinking that at the moment we're not checking the email and school match in any way? If so, could we have a ticket for adding that, and a TODO against that ticket?

Reply:
We might not know how to do it just yet, but it's important we don't forget 😛

Reply:
Yes, we are no longer matching schools and user emails. Happy to add a ticket but I don't think I need a todo as well. For the broken links etc, I agree that can be taken care of with a todo but for a feature, I would rather just have the ticket.

Reply:
Ok, sure